flow/project.py

```diff
@@ -4518,12 +4543,13 @@ def _select_jobs_from_args(self, args):
             # aggregates must be a set to prevent duplicate entries
             aggregates = set()
             for id in args.job_id:
```

Comment:
Change this variable name (perhaps `job_id`) so that it doesn't shadow the built-in function `id`.